Source/WebCore/animation/ScrollTimeline.cpp

```diff
@@ -102,6 +102,34 @@ ScrollTimeline::ScrollTimeline(Scroller scroller, ScrollAxis axis)
     m_scroller = scroller;
 }
 
+Element* ScrollTimeline::source() const
+{
+    switch (m_scroller) {
+    case Scroller::Nearest: {
+        if (CheckedPtr subjectRenderer = m_source->renderer()) {
+            if (RefPtr source = subjectRenderer->enclosingScrollableContainer()->element()) {
```

Comment:
You should null-check the result of `enclosingScrollableContainer()`.

Reply:
Presumably we can `ASSERT` so I'll add one.